src/lib/tools.ts

```diff
@@ -46,3 +72,107 @@ export async function writeYaml(
 	});
 	return obj;
 }
+
+export function gitURI(uri: string): boolean {
+	const re = new RegExp(
```

Comment:
```suggestion
	return /((git|ssh|http|https)|(git@[\w\.]+))(:(//)?)([\w\.@\:/\-]+)/.test(uri);
```

Reply:
This recommendation would yield a syntax error, as slashes need to be properly escaped here. I changed to this more dense style.